### ISSUE TITLE
RegEx Flag “s” is not supported

### ISSUE THREAD

**Issue by Jaid** [michaelgmcd/vscode-language-babel#103]
**Describe the issue**
When an inline RegEx is defined, using an s in the flags section breaks all the following code until the end of the file.

**Screenshot of Current Behavior**
Without flag:
![Without flag](https://i.imgur.com/05jp8c1.png)
With flag:
![With flag](https://i.imgur.com/kqUuJsw.png)
**Sample Code to Reproduce**
```js
import adbkit from "@devicefarmer/adbkit"
import {matches} from "super-regex"

export default class Device extends adbkit.DeviceClient {

  async getGpsAll() {
    const regex = /(?<provider>[a-z]+) provider.+?last location=Location.+?(?<lat>[\d.]+).+?(?<lon>[\d.]+).+?alt=(?<alt>[\d.]+)/gs
    const execution = await this.shell("dumpsys location")
    const responseBuffer = await adbkit.default.util.readAll(execution)
    const response = responseBuffer.toString("utf8")
    return matches(regex, response).map(match => match.namedGroups)
  }

  async getGps() {
    const gps = await this.getGpsAll()
    return gps[0]
  }

}
```

**Comment by Jaid** [comment]
TypeScript parser for comparison
![TypeScript parser for comparison](https://i.imgur.com/rY8BBuC.png)

**Comment by Tain101** [comment]
The 'd' flag has the same issue..

**Comment by michaelgmcd** [comment]
This should be fixed in 0.0.40
